webui/react/src/components/LogViewer.tsx

```diff
@@ -352,6 +352,7 @@ const LogViewer: React.FC<Props> = forwardRef((
       setTimeout(() => {
         if (!container.current || !scrollToInfo.logId) return;
         const top = config.messageSizes[scrollToInfo.logId].top;
+        setScrollToInfo({ isPrepend: false, logId: 0 });
```

Comment:
Basically the issue stems from the fact that the `isPrepend` does not reset upon scrolling, which is used to detect conditions for this "scroll to previous location" behavior. This line corrects it.